taskcluster/__init__.py

```diff
@@ -4,6 +4,7 @@
 import logging
 import os
 from .client import createSession  # NOQA
+from .client import createTemporaryCredentials  # NOQA
```

Comment:
I think this is the only change needed here. The `gen{Code,Docs}.py` changes appear to have broken the tests.

Reply:
..and that change has landed elsewhere.

Reply:
It appears *not* to have landed elsewhere.  I'll open a PR with just this single change.